resource_lists/astroneer/resources.yaml

```
@@ -2,531 +2,1697 @@
 authors:
   Asher: "https://aglick.com"
   Kevin: ""
+  Alban: "https://github.com/mopolita"
 
 index_page_display_name: "Astroneer"
 # row_group_count: 0
 
 recipe_types:
   Backpack Printer: "Use a Backpack Printer to turn {IN_ITEMS} into {OUT_ITEM}"
-  Smelter: "Use a Smelter to turn {IN_ITEMS} into {OUT_ITEM}"
-  Large Platform: "Use a Large Platform to turn {IN_ITEMS} into {OUT_ITEM}"
-  Printer: "Use a Printer to turn {IN_ITEMS} into {OUT_ITEM}"
-  Vehicle Bay: "Use a Vehicle Bay to turn {IN_ITEMS} into {OUT_ITEM}"
-  Printer and Vehicle Bay: "Use a Printer and Vehicle Bay to turn {IN_ITEMS} into {OUT_ITEM}"
+  Small Printer: "Use a Small Printer to turn {IN_ITEMS} into {OUT_ITEM}"
+  Medium Printer: "Use a Medium Printer to turn {IN_ITEMS} into {OUT_ITEM}"
+  Large Printer: "Use a Large Printer to turn {IN_ITEMS} into {OUT_ITEM}"
+  Smelting Furnace: "Use a Smelting Furnace to smelt {IN_ITEMS} into {OUT_ITEM}"
+  Chemistry Lab: "Use a Chemistry Lab to turn {IN_ITEMS} into {OUT_ITEM}"
+  Soil Centrifuge: "Use a Soil Centrifuge to make {OUT_ITEM} from {IN_ITEM}"
+  Trade Platform: "Trade {IN_ITEM} for {OUT_ITEM}"
 
 resources:
-  Coal:
+  Soil:
     recipes:
     - output: 1
       recipe_type: Raw Resource
+
+  Compound:
+    recipes:
+    - recipe_type: Raw Resource
+    - output: 4
+      recipe_type: Soil Centrifuge
+      requirements:
+        Soil: -1
+    - output: 2
+      recipe_type: Trade Platform
+      requirements:
+        Scrap: -1
+
+  Resin:
+    recipes:
+    - recipe_type: Raw Resource
+    - output: 4
+      recipe_type: Soil Centrifuge
+      requirements:
+        Soil: -1
+    - output: 2
+      recipe_type: Trade Platform
+      requirements:
+        Scrap: -1
+
+  Organic:
+    recipes:
+    - recipe_type: Raw Resource
+        - output: 4
```

I get a yaml parse error here

```
yaml.scanner.ScannerError: mapping values are not allowed here
  in "resource_lists/astroneer/resources.yaml", line 53, column 17
```
Yaml cares about indentation so it is probably getting tripped up with this array element not being aligned with the other elements under `recipes` and just needs to be unintended. Let me know if you are not receiving this error on your end just in case there is a bug in the builder code (I have already found one I need to fix).

```suggestion
    - output: 4
```